student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -203,4 +213,35 @@ public void approveAccommodations(UUID examId, ApproveAccommodationsRequest appr
     
     return Optional.absent();
   }
+  
+  @Override
+  public Response<ExamConfiguration> startExam(UUID examId) throws ReturnStatusException {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+    headers.setContentType(MediaType.APPLICATION_JSON);
+    HttpEntity<?> requestHttpEntity = new HttpEntity<>(headers);
+    Response<ExamConfiguration> response;
+    UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(String.format("%s/%s/start", examUrl, examId));
```

Comment:
It probably isn't a huge issue for us today, but I agree we should probably instead use the [buildAndExpand](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/util/UriComponentsBuilder.html#buildAndExpand-java.lang.Object...-) feature.